skrub/tests/test_dropnulls.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+import pandas as pd
+import polars as pl
+import pytest
+
+from numpy.testing import assert_array_equal
+
+from skrub import TableVectorizer
+from skrub._dataframe import _common as ns
+from skrub._dropnull import DropNullColumn
+
+
+@pytest.fixture
+def main_table(df_module):
+    return df_module.make_dataframe(
+        {
+            "idx": [
+                1,
+                2,
+                3,
+            ],
+            "value_nan": [
+                np.nan,
+                np.nan,
+                np.nan,
+            ],
+            "value_null": [
+                None,
+                None,
+                None,
+            ],
+        }
+    )
+
+
+def test_drop_nullcolumn(main_table):
```

Comment:
All the tests in this file are going to test DropNullColumn so I'd avoid this test name